eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/service/MeetingRoomService.java

```diff
@@ -0,0 +1,48 @@
+package hu.flowacademy.meetingorganizer.service;
+
+import hu.flowacademy.meetingorganizer.persistence.model.MeetingRoom;
+import hu.flowacademy.meetingorganizer.persistence.repository.MeetingRoomRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+@Transactional
+public class MeetingRoomService {
+
+    private MeetingRoomRepository meetingRoomRepository;
+
+    public List<MeetingRoom> findAllMeetingRooms () {
+        return meetingRoomRepository.findAll();
+    }
+
+    public MeetingRoom findOneMeetingRoomById (Long id) {
+        return meetingRoomRepository.findById(id).orElseThrow(()-> new ResponseStatusException(HttpStatus.NOT_FOUND));
+    }
+
+    public ResponseEntity<?> createMeetingRoom (MeetingRoom meetingRoom) {
```

Comment:
Itt se legyen ? generikus

Reply:
ezt is :)

Reply:
továbbá a srácokkal az beszéltük meg, hogy mindenhol response entity-t kellenne visszaadni, ezért végeztem még máshol is átalakításokat.